src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -322,36 +318,65 @@ module.exports = class ListObjectsCommandGenerator extends BaseCommandGenerator
 			return;
 		}
 
-		const importedObjects = data.customObjects.filter(
-			customObject => customObject.result.code === IMPORT_0BJECT.SUCCESS
-		);
-		const unImportedObjects = data.customObjects.filter(
-			customObject => customObject.result.code === IMPORT_0BJECT.FAILED
-		);
-
-		if (importedObjects.length) {
-			NodeUtils.println(
-				TranslationService.getMessage(MESSAGES.IMPORTED_OBJECTS),
-				NodeUtils.COLORS.RESULT
-			);
-			importedObjects.forEach(customObject =>
-				NodeUtils.println(
-					`${customObject.type}:${customObject.id}`,
-					NodeUtils.COLORS.RESULT
-				)
-			);
+		this._logImportedObjects(data.successfulImports);
+		this._logUnImportedObjects(data.failedImports);
+	}
+
+	_logImportedObjects(importedObjects) {
+		if (Array.isArray(importedObjects) && importedObjects.length) {
+			NodeUtils.println(TranslationService.getMessage(OUTPUT.IMPORTED_OBJECTS), NodeUtils.COLORS.RESULT);
+			importedObjects.forEach(objectImport => {
+				NodeUtils.println(`    - ${objectImport.customObject.type}:${objectImport.customObject.id}"`, NodeUtils.COLORS.RESULT);
+				this._logReferencedFileImportResult(objectImport.referencedFileImportResult);
+			});
 		}
-		if (unImportedObjects.length) {
-			NodeUtils.println(
-				TranslationService.getMessage(MESSAGES.UNIMPORTED_OBJECTS),
-				NodeUtils.COLORS.WARNING
-			);
-			unImportedObjects.forEach(customObject =>
+	}
+
+	_logReferencedFileImportResult(referencedFileImportResult) {
+		const importedFiles = referencedFileImportResult.successfulImports;
+		const unImportedFiles = referencedFileImportResult.failedImports;
+
+		if (!Array.isArray(importedFiles) && !Array.isArray(unImportedFiles)) {
+			return;
+		}
+
+		if (importedFiles.length || unImportedFiles.length) {
+			const referencedFilesLogMessage = `        - ${TranslationService.getMessage(OUTPUT.REFERENCED_SUITESCRIPT_FILES)}`;
+			NodeUtils.println(referencedFilesLogMessage, NodeUtils.COLORS.RESULT);
+
+			importedFiles.forEach(importedFile => {
+				const importedFileLogMessage = `            - ${TranslationService.getMessage(
```

Comment:
Also here, it would we awesome to do the tabs with \t characters... I think that way it would be more clear the identation of the message (I think in this one the identation is double tab no?)

Reply:
I understand. My concern is, is it possible tab sizes may differ in different platform/terminal? If so, our output may become inconsistent. Using spaces I think will ensure our indentation will be consistent across different platforms/terminal. I don't have a strong opinion about this, I'm not really sure. @margabit what are your thoughts?

Reply:
I think the user won't notice, and if he does, he has always the opportunity to change his configuration.
On the other hand, we can use then \b backspace character, if this is a problem. Or a Constant String called something like IDENTATION_2_SPACES,, IDENTATION_4_SPACES or so on.
I think we risk of setting different identations for the messages.

Reply:
Your suggestion of using a constant is a very good idea, I'l use that.